### ISSUE TITLE
why not just use F.softmax(x, dim=1), is this function not the same effect?

### ISSUE THREAD

**Issue by hackerekcah** [kaushalshetty/Structured-Self-Attention#9]
https://github.com/kaushalshetty/Structured-Self-Attention/blob/9c91df83e76d8d898a0ebc03360179154bbb5195/attention/model.py#L73

**Comment by kaushalshetty** [comment]
Yes it should. Softmax did not support this when this code was developed. But now it does.
